meditool/data_types/signal/features.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import scipy.io.wavfile
+import matplotlib.pyplot as plt
+
+
+
+
+def turns(fp: int,signal :np.ndarray ,const: float) -> int:
+    """Method for counting number of turns in signal.
+    :param fp: sampling rate
+    :param signal:  numpy.ndarray with signal values
+    :param const: parameter that defines multiple of standard deviation. This multiple is a minimum value of signal change which function interprets as a turn.
```

Comment:
Zmieniłbym nazwę argumentu na bardziej znaczącą, czyli np `minimal_signal_change`

Reply:
Czy wtedy zachować formę, czyli parametr w zależności od odchylenia standardowego, czy po prostu ma to być wartość bezwzględna określająca jak duża ma być minimalna zmiana w sygnale?